docs/introduction.rst

```diff
@@ -241,6 +241,35 @@ The ``citation`` command show the citation for the retriever and for the scripts
 
 **To create new, edit, delete scripts please read the documentation on scripts**
 
+
+Storing database connection details
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The retriever reads from the standard configuration files for the database
+management systems. If you want to store connection details they should be
+stored in those files. Make sure to secure these files appropriately.
+
+For postgreSQL, create or modify `~/.pgpass`. This is a file named `.pgpass`
+located in the users home directory. It should take the general form:
+
+``hostname:port:database:username:password``
+
+where each word is replaced with the correct information for your database
+connection or replaced with an ``*`` to apply to all values for that section.
+
+For MySQL, create or modify `~/.my.cnf`. This is a file named `.my.cnf` located
+in the users home directory. The relevant portion of this file for the retriever
+is the `client` section which should take the general form:
+
+::
+
+   [client]
+   host=localhost
+   port=3306
```

Comment:
Good call. I just changed them to match the same wording as for postgres and added an equivalent paragraph describing what to do with them.